src/openrct2/object/ObjectRepository.cpp

```diff
@@ -101,7 +101,7 @@ class ObjectFileIndex final : public FileIndex<ObjectRepositoryItem>
     }
 
 public:
-    std::tuple<bool, ObjectRepositoryItem> Create(sint32 language, const std::string &path) const override
+    std::tuple<bool, ObjectRepositoryItem> Create([[maybe_unused]] sint32 language, const std::string& path) const override
```

Comment:
`language` is unused, should be dropped

Reply:
This overrides the abstract member function of `FileIndex`. It can only be dropped by changing that base class, which is out of scope for this PR. I don't know with what intentions this argument was added, so I'd rather leave it as is.

Reply:
This is actually why I don't think this unused argument warning is very useful. Any function that implements an interface will not necessarily need to use all its arguments.

Reply:
I think it's still better than omitting the argument name completely, so that it's at least clear what data it holds. Shall I change these to `/*language*/` then, just like I did for dummy classes?

Reply:
The reason for enabling such warning would be to go in and inspect the (lack of) use of argument. If we were to just blindly apply it for sake of having another warning, maybe it's better to not have it at all.

I agree not all functions implementing an interface have to use all arguments, but if none of the implementations do, what's the point for argument in the first place?